src/cascadia/TerminalApp/lib/pch.h

```diff
@@ -36,6 +36,7 @@
 #include <winrt/Windows.UI.Xaml.Hosting.h>
 #include "winrt/Windows.UI.Xaml.Markup.h"
 #include "winrt/Windows.UI.Xaml.Documents.h"
+#include <winrt/Windows.ApplicationModel.DataTransfer.h>

```

Comment:
DataTransfer necessary?

Reply:
Yea we need this to access the clipboard (because _obviously_, right?)